geometry-1/DOMRect-001.html

```diff
@@ -60,7 +60,7 @@
         },'testConstructorNegativeWidthHeight');
         test(function() {
             checkDOMRect(new DOMRect(0, 0, undefined, 4),
-                { x: 0, y: 0, width: NaN, height: 4, top: 0, right: 0, bottom: 4, left: 0 });
+                { x: 0, y: 0, width: 0, height: 4, top: 0, right: 0, bottom: 4, left: 0 });

```

Comment:
I think the default value should be used here, see [4.5.1.1., step 10.4.](http://heycam.github.io/webidl/#es-interface-call).